src/App.js

```diff
@@ -14,11 +15,64 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+export class App extends React.Component {
+  state = {
+    selected: [],
+  };
 
-export default App;
+  handleClick = (e) => {
+    const { selected } = this.state;
+    const good = e.target.textContent;
+
+    if ((e.ctrlKey || e.metaKey) && !selected.includes(good)) {
+      this.setState({ selected: [...selected, good] });
+    } else if ((e.ctrlKey || e.metaKey)
+    && selected.includes(good)) {
+      this.setState({ selected: selected.filter(item => item !== good) });
+    } else {
+      this.setState({ selected: [good] });
+    }
+  }
```

Comment:
виконання бомба, дам тільки пораду як краще форматувати код
```suggestion
    if (
      (e.ctrlKey || e.metaKey)
        && !selected.includes(good)
    ) {
      this.setState({ selected: [...selected, good] });
    } else if (
      (e.ctrlKey || e.metaKey)
        && selected.includes(good)
    ) {
      this.setState({ selected: selected.filter(item => item !== good) });
    } else {
      this.setState({ selected: [good] });
    }
  }
```